scripts/generators/asciidoc_fields.py

```diff
@@ -183,6 +183,9 @@ def index_row():
 
 def index_footer():
     return '''
+
```

Comment:
@lcawl I wanted to add a `[float]` here to but get errors about unexpected title levels if I do this. My goal was to have the overview table also on the same page as all the fields.

Reply:
> My goal was to have the overview table also on the same page as all the fields.

Great idea!